Libraries/Text/TextStylePropTypes.js

```diff
@@ -49,7 +49,16 @@ const TextStylePropTypes = {
   textShadowRadius: ReactPropTypes.number,
   textShadowColor: ColorPropType,
   /**
-   * @platform ios
+   * Increase or decrease the spacing between characters. The default value is 0, for no extra
```

Comment:
Can you add these docs to the docs at https://github.com/facebook/react-native-website instead? We'll be slowly pruning out these comment blocks over the next few months.

Reply:
(It's fine to wait for this PR to be merged first)